mocks/path.js

```diff
@@ -0,0 +1,42 @@
+class Path {
+  constructor(segments) {
+    this.segments = segments;
+  }
+  compareTo(other) {
```

Comment:
There is a lot of logic in this function.  Can we get a test asserting each of these cases?

Reply:
Thanks for reply. I've just added tests for  `compareTo` function